src/services/votes.spec.ts

```diff
@@ -331,27 +330,11 @@ describe('service: votes', () => {
       otherGroupCategory!.id,
     ]);
 
-    expect(groups).toBeDefined();
-    expect(groups['unexpectedKey']).toBeUndefined();
-    expect(typeof groups).toBe('object');
-    expect(Object.keys(groups).length).toEqual(2);
-    expect(groups[Object.keys(groups)[0]!]!.length).toEqual(2);
-  });
-
-  test('only return baseline groups for users who voted for the option as non of the users is in the additional group category', async () => {
-    // Get vote data required for groups
-    const voteArray = await queryVoteData(dbPool, questionOption?.id ?? '');
-    const votesDictionary = await numOfVotesDictionary(voteArray);
-    const groups = await groupsDictionary(dbPool, votesDictionary, [
-      groupCategory!.id,
-      unrelatedGroupCategory!.id,
-    ]);
-
-    expect(groups).toBeDefined();
-    expect(groups['unexpectedKey']).toBeUndefined();
-    expect(typeof groups).toBe('object');
-    expect(Object.keys(groups).length).toEqual(1);
-    expect(groups[Object.keys(groups)[0]!]!.length).toEqual(2);
```

Comment:
Why did this test get deleted? I think it is a very usefull test to assert that only relevant group categories for the plurality score calculation are considered.

Reply:
i agree, but this test does not actually test that.

it tests that there is only one group that voted and that 2 users voted.

this test should be re created where it tests that the groups we receive are the groups that the users voted on and the only ones we expect. the number testing is all good while our seeds are only for this test but since it is global it makes it super hard to actually tell what is going on here once 1 thing in the seed changes (our biggest kryptonite)

Reply:
okey makes sense. So lets exclude it.